assets/filter-icon.svg

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Is there a reason why we want to use an svg as opposed to an icon library?

Reply:
no, I just don't know any icon libraries lmao, have any recommendations?